lib/crawler.js

```diff
@@ -11,53 +10,55 @@ var path = require('path')
 	, Bottleneck = require('bottleneckp')
 	, seenreq = require('seenreq')
 	, iconvLite = require('iconv-lite')
-	, typeis = require('type-is').is;
+	, typeis = require('type-is').is
+	, http2 = require('http2')
```

Comment:
http2 is added in version 8 and is stable in version 10, let's do some polyfill.

Reply:
Version | Changes
-- | --
v10.10.0 | HTTP/2 is now Stable. Previously, it had been Experimental.
v8.4.0 | Added in: v8.4.0